tssc/__init__.py

```diff
@@ -231,15 +230,23 @@
           #password: None
         }
 
-      security-static-code-analysis:
-      # WARNING: not yet implemented
+      static-code-analysis:
       - implementer: SonarQube
-        config: {}
+        config: {
+          # Required.
+          # URL to the sonarqube server
+          #url: 'http//sonarqube-sonarqube.company.com/'
```

Comment:
We've been uncommenting the required parameters

Reply:
updated